indigo_app/templates/place/detail.html

```diff
@@ -93,7 +103,12 @@ <h5 class="mb-0">Current tasks ({{ total_open_tasks }})</h5>
             {% if item.count %}
               <div class="row mb-2">
                 <div class="col-4">
-                  <i class="fas fa-sm fa-fw task-icon-{{ item.state }}"></i> {{ item.state_string }}
+                  <i class="fas fa-sm fa-fw task-icon-{{ item.state }}"></i> 
+                  {% if item.state == 'open' %}
+                    <a class="text-dark" href="{% url 'tasks' place=place.place_code %}"">{{ item.state_string }}</a>
+                  {% else %}
+                    <a class="text-dark" href="{% url 'tasks' place=place.place_code %}?state={{ item.state }}&format=columns">{{ item.state_string }}</a>
```

Comment:
in other words, I would just keep this – if they want to see all tasks there's already a link for that in the card header

Reply:
Ah, I get your point now. The requirements stated that the `open` should link to the normal task listing page direct, but I prefer yours